api/src/main/java/org/openmrs/validator/PatientProgramValidator.java

```diff
@@ -73,9 +73,7 @@ public boolean supports(Class<?> c) {
 	 */
 	@Override
 	public void validate(Object obj, Errors errors) {
-		if (log.isDebugEnabled()) {
-			log.debug(this.getClass().getName() + ".validate...");
```

Comment:
Can you take a second look at this?

Reply:
@gracebish did you see the above comment?

Reply:
@gracebish did you see the above comment?

Reply:
@dkayiwa, i saw the comment. I have critically analyzed the issue but lost completely. Could you kindly throw more light?

Reply:
Why are you passing two string constants instead of one, in the log.debug method?

Reply:
In the case where i had put {} {}, one was meant for the method name and the other for the class.

Reply:
And for this case?

Reply:
It should be log.debug("{}.validate...", this.getClass().getName());
In case i remove the if (log.isDebugEnabled()), will the code be executed ?

Reply:
For the previous pull requests, where you removed it, was the code getting executed?